src/pages/AppDetails/AppInfo/AppDescription.tsx

```diff
@@ -22,7 +22,7 @@ class AppDescription extends React.Component<AppDescriptionProps, AppDescription
   }
 
   istioSidecar() {
-    let istioSidecar = true;
+    let istioSidecar = this.props.app.workloads && this.props.app.workloads.length > 0 ? true : false;
```

Comment:
(not for this PR but...) it's strange that we duplicate the same algorithm, to check for sidecar on the backend side for app list, and on the frontend side for app details, is there a reason for that?

Reply:
Oh, true, the istioSide car check is done in the backend for the list, but it was not added in the details of the app.
I can change it on both (backend/frontend).